scripts/rollup/plugins.js

```diff
@@ -144,6 +144,7 @@ const terserMinified = terser({
   ie8: false,
   toplevel: true,
   compress: {
+    keep_fnames: true,
```

Comment:
This is only used for the production bundle, so this should be reverted 😅 also, Closure is responsible for most of the function name mangling (in both production or development) and this option is turned off on Terser in development